src/lib/AdminSidebar.js

```diff
@@ -53,8 +63,16 @@ class AdminSidebar extends React.Component {
           <div key={idx} style={{ marginBottom: '25px' }}>
             <NavSectionTitle secondary>{categoryName}</NavSectionTitle>
 
-            {widgets.filter(widgetName => library[widgetName].category === categoryName).sort().map((widgetName, widgetIdx) =>
-              <SidebarItem key={widgetIdx} onClick={() => onAddWidget(widgetName)}>{library[widgetName].title}</SidebarItem>
+            {widgets.filter(widgetName => library[widgetName].category === categoryName).map((widgetName) =>
+              ({ widgetName, widgetTitle: library[widgetName].title })).sort((a, b) =>
```

Comment:
will calling directly `.sort(sortByTitle)` instead of `.sort((a, b) => sortByTitle(a,b))` work here?